index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta name="viewport" content='width=device-width, initial-scale=1.0' />
+    <title>adopt a dog</title>
+    <link rel="stylesheet" type="text/css" href="styles.css"/>
+    <script src="script.js"></script>
+  </head>
+  <body>
+  <!-- Header -->
+<header>
+  <!-- Left Side-->
+  <div class="banner-left-side">
+    <img src="images/adoptadog-logo.svg" alt="logo" class="banner-left-side-img"/>
+    <h3 style="color:#8bc34a;font-size:20px;">
+      <i>Seattle's premiere dog adoption agency</i>
+    </h3>
+  </div>
+  <!-- Right Side-->
+  <div class="banner-right-side">
+    <div class="banner-right-side-cart">
+      <img src="images/cart-icon.svg" class="cart" alt="cart"/>
+      <strong>
+        <p class="cart-price">$<span id="demo"></span></p>
+      </strong>
+    </div>
+    <div>
+      <a href="Page-4.html" class="button">
+          Checkout
+        </a>
+    </div>
+  </div>
+</header>
+  <!-- Nav bar -->
+  <nav>
+    <h4 class="nav-caption">
+      <a href="index.html"> Home </a>/
+      <a href="Page-2.html"> Dogs </a>/
+      <a href="Page-3.html"> Blog </a></h4>
+  </nav>
+<div class="container">
+  <!-- Main Container -->
+  <div class="main-container">
+    <section>
+      <div><img src="images/home-intro.jpg" alt="home"/></div>
```

Comment:
This image needs a percentage size, so it resizes correctly and does not interrupt your styles.